src/server/routes.ts

```diff
@@ -22,10 +22,6 @@ router.post('/plants', async (req, res, next) => {
 router.get('/plants', async (_req, res, next) => {
   try {
     const plants = await findPlants();
-    if (!plants) {
```

Comment:
Are you sure you don't need this? 🙃

Reply:
Yes, because it will be always given me an empty array back and never nothing.